tests/components/Help.test.tsx

```diff
@@ -0,0 +1,10 @@
+import { shallow } from 'enzyme';
+import React from 'react';
+import Help from '../../src/components/Help';
+
+describe('Help', () => {
+    it('should render the Help component', () => {
+        const wrapper = shallow(<Help />);
+        expect(wrapper).toMatchSnapshot();
+    });
+});
```

Comment:
Should we have a test to check that the Help component isn't rendered when the contact page is being shown?

Reply:
yeah can add this